### ISSUE TITLE
possibility to specify only released sources

### ISSUE THREAD

**Issue by santiagozky** [lardbit/nefarious#215]
It would be great if we could specify to get only movies after they've been officially released on WEB/bluray to avoid getting screeners or steadycam.

it could be similar to what flexget already does where you can set a[ minimum quality](https://www.flexget.com/Plugins/quality) not only for resolution for also for source or audio

**Comment by lardbit** [comment]
Good idea.  I'll roll this into the "dynamic quality profile" PR i'm working on:  https://github.com/lardbit/nefarious/pull/212
